### DISCUSSION TITLE
Neo Launcher features merge request

### DISCUSSION THREAD

**Discussion by Niavy64** [LawnchairLauncher/lawnchair#4598] [Ideas]
Hi,
I am a longtime user of Nova Launcher, then I switched 1 year ago to Neo Launcher, but this great launcher is discontinued. I am still using it and installing it, but I am also searching for a good replacement launcher. Lawnchair is the closest I found until now, layout and icon internal style along with Arcticons completion are great, but compared to Neo, it lacks two features I use very often :
- Tabs in drawer : I separate "basic stock-like phone apps", "downloaded apps" and "entertainment apps & games". I see no such option in Lawnchair
- "Cover mode" : homescreen folders can appear as a single icon (folder's first icon) instead of folder content small icons. When you tap this single icon, you launch corresponding app. When you swipe up on this icon, you open corresponding folder.
Simpler cover mode option if gesture is complicated to integrate : simple click launches first folder app, long click opens folder and three-dot option menu

Can these two features be added to Lawnchair ? This definitely would make me switch to it.
.
...oh and a layout save feature also.

**Comment by tungstengmd** [comment]
just here to let you know that neo isn't discontined, it still receives commits and new builds can be downloaded via actions, like lawnchair.

**Reply by Niavy64** [reply]
> just here to let you know that neo isn't discontined, it still receives commits and new builds can be downloaded via actions, like lawnchair.

Thank you. Yet development does not seem very active. That's too bad, this is the best launcher I've tried so far.

**Comment by SephGER** [comment]
The cover mode is available in Lawnchair 2, will it be ported someday?
